src/main/java/pl/adam/BookManager.java

```diff
@@ -22,43 +22,39 @@ public void displayList() {
         }
     }
 
-    public void searchBookByTitle(String bookName) {
-        boolean isThere = false;
+    public ArrayList<String> searchBookByTitle(String bookName) {
+        ArrayList<String> titleList = new ArrayList<>();
         for (Book book : bookList) {
             if (book.getTitle().equalsIgnoreCase(bookName)) {
-                System.out.println(book);
-                isThere = true;
+                titleList.add(book.toString());
             }
         }
-        if (!isThere){
-            System.out.println("Nie znaleziono książki o podanym tytule");
+        if (!titleList.isEmpty()) {
+            return titleList;
```

Comment:
It's supposed to return objects of Book, not just a title (search book (object) by title (string))